imagepipeline/src/main/java/com/facebook/imagepipeline/request/ImageRequestBuilder.java

```diff
@@ -27,6 +30,12 @@
  */
 public class ImageRequestBuilder {
 
+  /**
+   * Local resource supported extensions
+   */
+  private static final String LOCAL_RESOURCE_EXT_PNG = ".png";
+  private static final String LOCAL_RESOURCE_EXT_JPG = ".jpg";
```

Comment:
Android supports gifs too, although these are discouraged. You could simply put these into a HashSet instead.